lib/pymedphys/_experimental/chartchecks/helpers.py

```diff
@@ -0,0 +1,417 @@
+# Copyright (C) 2018 Cancer Care Associates
```

Comment:
Can you change this copyright to your own, there's not much left from the original file here.

Also, this license header, once you put your copyright on it, can you put this same header at the top of all of your files?